package.json

```diff
@@ -5,7 +5,7 @@
 	"displayName": "MongoDB",
```

Comment:
This looks good, but none of the changes show up in the VS Code marketplace. What if we add `[DEPRECATED]` to the 'displayName' and 'description', bump the version, and republish?